rpc/jsonrpc/internal_api.go

```diff
@@ -0,0 +1,65 @@
+package jsonrpc
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/erigontech/erigon-lib/kv"
+	"github.com/erigontech/erigon-lib/kv/rawdbv3"
+	"github.com/erigontech/erigon/turbo/snapshotsync/freezeblocks"
+)
+
+// Defines the `internal_` JSON-RPC namespace.
```

Comment:
we have:
- `go run ./cmd/diag dbs`: require working Erigon node
- `integration print_stages`: doesn't require working Erigon (but doesn't say anything about "current Erigon status")
- `internal_` JSON-RPC: can work without erigon (external rpcd), and with Erigon (or inside Erigon). Interesting compromises... 

Don't know what is better :-)

Reply:
I think `internal_` could be a preferred way for erigon devs to hook quick and dirty querying stuff.

`integration <blablabla>` would work better for generating long running reports (RPC would suffer from HTTP timeout) and/or more interactive hacks

but both would be for our internal use, as opposed to diagnostics I guess